HVSample/src/main/java/com/microsoft/hsg/android/hvsample/PersonalImageLoader.java

```diff
@@ -33,80 +33,95 @@
 
 public class PersonalImageLoader implements ComponentCallbacks2 {
 
-	private ImageLruCache cache;
-	private HealthVaultClient hvClient;
-	private Activity context;
+	private ImageLruCache mCache;
+	private HealthVaultClient mClient;
+	private Activity mContext;
 	
 	private ImageView imageView;
 	
 	public PersonalImageLoader(Activity context,
 			HealthVaultClient client) {
 		ActivityManager am = (ActivityManager) context.getSystemService(Context.ACTIVITY_SERVICE);
-        int memSize = am.getMemoryClass() * 1024 * 1024;
-        
-        cache = new ImageLruCache(memSize);
-        hvClient = client;
-        this.context = context;
+		int memSize = am.getMemoryClass() * 1024 * 1024;
+
+		mCache = new ImageLruCache(memSize);
+		mClient = client;
+		mContext = context;
```

Comment:
Need to be careful when holding references to contexts. We pass in an Activity context to this class and this class performs long running tasks (network requests). It is possible that the activity could be killed while this task in running, but the system won't be able to free up the resources for the activity because we are holding a reference to its context here. In this case it doesn't really seem like a big issue since the request will finish and then it should be destroyed once it goes idle and the activity dies. But context handling is something we need to be careful about and even in this situation it would be beneficial to hold WeakReference to the context instead as to not interfere with system garbage collection.